src/styles/index.css

```diff
@@ -20,3 +38,36 @@ header .buttons-become_collab {
 	background-color: white;
 	border-bottom: solid 1px black;
 }
+
+footer {
+	width: 100%;
```

Comment:
A footer element by default has `width: 100%;`. I think that line is innecesary.

Reply:
Removed on next commit